tracecat/actions/integrations/iam/okta.py

```diff
@@ -28,6 +28,49 @@
 """
 
 
+def create_okta_client() -> httpx.AsyncClient:
+    OKTA_API_TOKEN = os.getenv("OKTA_API_TOKEN")
+    if OKTA_API_TOKEN is None:
+        raise ValueError("OKTA_API_TOKEN is not set")
+    client = httpx.AsyncClient(
+        base_url=os.getenv("OKTA_BASE_URL"),
+        headers={
+            "Authorization": f"SSWS {OKTA_API_TOKEN}",
+            "Accept": "application/json",
+            "Content-Type": "application/json",
+        },
+    )
+    return client
+
+
+@registry.register(
+    default_title="Find Okta User",
+    description="Find an Okta user by login username or email",
+    display_group="Okta",
+    namespace="integrations.okta",
+    secrets=[okta_secret],
+)
+async def find_okta_user(
+    username_or_email: Annotated[
+        str,
+        Field(..., description="Login username or e-mail to find"),
+    ],
+) -> list:
+    async with create_okta_client() as client:
+        params = [
+            "search",
+            f"profile.login%20eq%20%22{username_or_email}%22%20or%20profile.email%20eq%20%22{username_or_email}%22",
+        ]
```

Comment:
This should be a dict like
```python
params = {
    "search": ...
}
```

Reply:
Noticed that when I was looking at this to remove the encoding, fixing.